package_manifest.yaml

```diff
@@ -1087,6 +1092,23 @@ repoclosures:
           - el8-powertools
           - el8-puppet-6
           - "el8-foreman-{{ foreman_version }}"
+    katello-repoclosure-el8:
+      repoclosure_target_repos:
+        el8:
+          - "el8-katello-{{ katello_version }}"
+      repoclosure_target_dist: el8
+      repoclosure_lookaside_repos:
+        el8:
+          - el8-baseos
+          - el8-appstream
+          - el8-powertools
```

Comment:
that means our users need to enable powertools too?

Reply:
That is unfortunately, correct. They need it for `libdb_cxx`.

Reply:
:crying_cat_face: but oh well, I don't want to carry another set of packages for that crazyness. is updating install/upgrade docs on the 4.0 "release table"?